cmd/kubeapps-apis/proto/kubeappsapis/plugins/fluxv2/packages/v1alpha1/fluxv2.proto

```diff
@@ -82,59 +79,13 @@ service FluxV2PackagesService {
   }
 }
 
-// Specific messages used by the 'fluxv2' plugin
-
-// GetPackageRepositories
-//
-// Request for GetPackageRepositories
-message GetPackageRepositoriesRequest {
-  // The context (cluster/namespace) for the request
-  kubeappsapis.core.packages.v1alpha1.Context context = 1;
-  // TODO: Add standard filters.
-}
-
-// GetPackageRepositories
-//
-// Response for GetPackageRepositories
-message GetPackageRepositoriesResponse {
-  option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
-    example: '{"repositories": [{"name": "repo-name.example.com", "namespace": "", "url": "foo.registry.example.com/repo-name/main@sha256:cecd9b51b1f29a773a5228fe04faec121c9fbd2969de55b0c3804269a1d57aa5"}]}'
-  };
-
-  // Repositories
-  //
-  // List of PackageRepository
-  repeated PackageRepository repositories = 1;
-}
-
-// PackageRepository
-//
-// A PackageRepository defines a repository of packages for installation.
-message PackageRepository {
-  // TODO: add example for API docs
-  // option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_schema) = {
-  //   example: '{}'
-  // };
-
-  // Package repository name
-  //
-  // The name identifying package repository on the cluster.
-  string name = 1;
-
-  // Package repository namespace
-  //
-  // An optional namespace for namespaced package repositories.
-  string namespace = 2;
-
-  // Package repository URL
-  //
-  // A url identifying the package repository location.
-  string url = 3;
-
-  // Package repository plugin
-  //
-  // The plugin used to interact with this package repository.
-  kubeappsapis.core.plugins.v1alpha1.Plugin plugin = 4;
-
-  // TODO: Other fields such as type, status... TBD.
+service FluxV2RepositoriesService {
+  // AddPackageRepository add an existing package repository to the set of ones already managed by the 
+  // 'fluxv2' plugin
+  rpc AddPackageRepository(kubeappsapis.core.packages.v1alpha1.AddPackageRepositoryRequest) returns (kubeappsapis.core.packages.v1alpha1.AddPackageRepositoryResponse) {
+    option (google.api.http) = {
+      post: "/core/packages/v1alpha1/repositories"
```

Comment:
The url here needs to be specific to the flux plugin like the others above?

Reply:
indeed. will fix